builtin/providers/triton/resource_machine.go

```diff
@@ -108,17 +109,53 @@ func resourceMachine() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
-			"networks": {
```

Comment:
Yes, see line 193. The code handles both but prints a warning because the field is marked as deprecated. I just added `nic` above `network`, which I guess shows up weird in this diff.